rewards/utils/emission_utils.py

```diff
@@ -32,6 +33,39 @@ def get_flat_emission_rate(sett: str, chain: str) -> Decimal:
     return Decimal(get_emission_control(chain).proRataEmissionRate(sett).call() / 1e4)
 
 
+@lru_cache
+def get_nft_control(chain: Network) -> ContractFunctions:
+    return make_contract(
+        EMISSIONS_CONTRACTS[chain]["NFTControl"], Abi.NFTControl, chain
+    )
+
+
+@lru_cache
+def get_nft_weights(chain: Network):
+    nft_control = get_nft_control(chain)
+    schedules = list(
+        map(
+            lambda ws: NFTWeightSchedule(ws[0], ws[1], ws[2], ws[3]),
```

Comment:
Readability on this is a bit tough.  Maybe add a comment here explaining what's going on with the `ws[n]` inputs to the `NFTWeightSchedule` for future developers who work on this or are reading it for reference.